packages/@webex/plugin-meetings/src/reconnection-manager/index.ts

```diff
@@ -385,6 +385,29 @@ export default class ReconnectionManager {
       'ReconnectionManager:index#executeReconnection --> Attempting to reconnect to meeting.'
     );
 
+    const wasSharing = this.meeting.shareStatus === SHARE_STATUS.LOCAL_SHARE_ACTIVE;
+
+    if (wasSharing) {
+      // Stop the share streams if user tried to reconnect
```

Comment:
very minor thing, but the comment is slightly misleading, because I don't think the user controls the reconnection, it's done automatically by SDK, isn't it?

Reply:
Yeah you're right. I'll remove this comment since I think with the new "stopLocalShareTrack" helper function it'll be pretty clear what this is doing.